webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/controllers/VolumeTracingController.scala

```diff
@@ -395,28 +398,37 @@ class VolumeTracingController @Inject()(
   private def getFallbackLayerDataIfEmpty(tracing: VolumeTracing,
                                           data: Array[Byte],
                                           missingBucketIndices: List[Int],
-                                          mag: String,
-                                          cxyz: String,
+                                          mag: Vec3Int,
+                                          position: Vec3Int,
+                                          cubeSize: Int,
                                           urlToken: Option[String]): Fox[Array[Byte]] = {
-
     def fallbackLayerData(): Fox[Array[Byte]] = {
       val organizationName = tracing.organizationName
       val dataSetName = tracing.dataSetName
       val dataLayerName = tracing.getFallbackLayer
+      val request = WebKnossosDataRequest(
+        position = Vec3Int(position.x * cubeSize * mag.x, position.y * cubeSize * mag.y, position.z * cubeSize * mag.z),
```

Comment:
this might be a good opportunity to extend the Vec3Int class so that it supports this directly as position * cubeSize * mag
https://stackoverflow.com/questions/17271270/is-it-possible-to-override-built-in-operators-in-scala